docs/tutorials/multimodal/object_detection/inference/detection_inference_quick_start.md

```diff
@@ -0,0 +1,177 @@
+# AutoMM Detection - Inference with Pretrained Faster R-CNN on VOC Format Dataset
```

Comment:
Later we can merge this into our quick start tutorial.

Reply:
yea for sure